src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -1,48 +1,41 @@
 package org.folio.rest.impl;
+
 import java.util.Map;
-import javax.ws.rs.core.Response;
-import org.folio.rest.jaxrs.model.TenantAttributes;
-import org.folio.rest.tools.utils.TenantLoading;
 
-import io.vertx.core.AsyncResult;
 import io.vertx.core.Context;
-import io.vertx.core.Handler;
-import io.vertx.core.Vertx;
-import io.vertx.core.logging.Logger;
-import io.vertx.core.logging.LoggerFactory;
+import io.vertx.core.Future;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.tools.utils.TenantLoading;
 
 public class TenantRefAPI extends TenantAPI {
- private static final Logger log = LoggerFactory.getLogger(TenantRefAPI.class);
 
+  private static final String SAMPLE_LEAD = "sample-data";
+  private static final String SAMPLE_KEY = "loadSample";
+  private static final String REFERENCE_KEY = "loadReference";
+  private static final String REFERENCE_LEAD = "ref-data";
```

Comment:
If these constants are only used in one place, why do they need to be defined in the scope of the entire class?

Personally, I find it easier to comprehend code when I need to navigate around less. I would prefer these were either moved to the only method they are used by or inlined.